thunder/tests/test_jit_general.py

```diff
@@ -629,7 +629,7 @@ def test_nanogpt():
 )
 @pytest.mark.parametrize(
     "device",
-    ("cpu", "cuda"),
+    ("cpu", "cuda", "meta"),
```

Comment:
Do we call PyTorch's meta functions anywhere in the pipeline? What's the state of meta tensors in PyTorch?